cmd/mothership/mothership/start/http.go

```diff
@@ -621,16 +621,39 @@ func sendResponse(w http.ResponseWriter, r *http.Request, clusterState *cluster.
 	}
 }
 
-func newClusterResponse(r *http.Request, clusterState *cluster.State) (*keb.HTTPClusterResponse, error) {
+func newClusterResponse(r *http.Request, clusterState *cluster.State, reconciliationRepository reconciliation.Repository) (*keb.HTTPClusterResponse, error) {
 	kebStatus, err := clusterState.Status.GetKEBClusterStatus()
 	if err != nil {
 		return nil, err
 	}
+
+	var failures []keb.Failure
+	reconciliations, err := reconciliationRepository.GetReconciliations(&reconciliation.WithRuntimeID{RuntimeID: clusterState.Cluster.RuntimeID})
+	if err != nil {
+		return nil, err
+	}
+	if len(reconciliations) > 0 {
+		operations, err := reconciliationRepository.GetOperations(reconciliations[0].SchedulingID)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, operation := range operations {
+			if operation.State.IsError() {
```

Comment:
good idea, but the name `isError` would indicate that we check only for the `[OperationStateError`](https://github.com/kyma-incubator/reconciler/blob/main/pkg/model/operationstate.go#L15). As we are interested in all operations which had Go-errors during their execution, I could imagine that the name `hasErrors()` is a bit better.